catalog/resource_recipient_test.go

```diff
@@ -52,11 +57,17 @@ func TestCreateRecipient(t *testing.T) {
 		comment = "b"
 		authentication_type = "TOKEN"
 		sharing_code = "c"
+		owner = "InitialOwner"
 		ip_access_list {
 		   allowed_ip_addresses = ["0.0.0.0/0"]
 		}
 		`,
-	}.ApplyNoError(t)
+	}.Apply(t)
```

Comment:
`ApplyAndExpectData` could be easier to use: https://github.com/databricks/terraform-provider-databricks/blob/master/qa/testing.go#L269